app/src/app/[lng]/data/[step]/SubsectorDrawer.tsx

```diff
@@ -493,51 +501,81 @@ export function SubsectorDrawer({
+                                  lineHeight="24px"
+                                  color="content.alternative"
+                                >
+                                  {t("scope")}{" "}
+                                  {scope.gpcReferenceNumber?.split(".").pop()}
+                                </Heading>
+                                <Text
+                                  color="content.tertiary"
+                                  fontSize="16px"
+                                  fontWeight="normal"
+                                  letterSpacing="wide"
+                                  fontStyle="normal"
+                                >
+                                  {scope.label}
+                                </Text>
                                 {isScopeCompleted(scope.value, scopeData) ? (
-                                  <Tag variant="success" mx={6}>
-                                    {t("completed")}
-                                  </Tag>
+                                  ""
                                 ) : (
-                                  <Tag variant="warning" mx={6}>
-                                    {t("incomplete")}
-                                  </Tag>
+                                  <Text
+                                    color="sentiment.negativeDefault"
+                                    size="body.md"
+                                    fontWeight="normal"
+                                    letterSpacing="wide"
+                                    lineHeight="20px"
+                                  >
+                                    {t("save-missing-scope-info")}
+                                  </Text>
                                 )}
-                                <AccordionIcon
-                                  borderWidth={1}
-                                  boxSize={6}
-                                  borderRadius="full"
-                                  borderColor="border.overlay"
-                                />
-                              </HStack>
-                            </AccordionButton>
-                          </h2>
-                          <AccordionPanel pt={4}>
+                              </Box>
+
+                              <AccordionIcon
+                                borderWidth={1}
+                                boxSize={6}
+                                borderRadius="full"
+                                borderColor="border.overlay"
+                              />
+                            </HStack>
+                          </AccordionButton>
+                        </h2>
+                        <AccordionPanel
+                          bg="background.backgroundGreyFlat"
+                          zIndex="9999"
+                          position="relative"
+                        >
+                          <form className="space-y-6 grow flex flex-col">
```

Comment:
Since there are multiple forms now, does the save logic need to be adjusted?
We can probably clean up and refactor the form state in a future PR though to not include the unused keys and get rid of the nested fields...